.travis.yml

```diff
@@ -0,0 +1,104 @@
+# Travis CI (MIT License) configuration file for Options Importer
+# @link https://travis-ci.org/
+
+# Declare project language.
+# @link http://about.travis-ci.org/docs/user/languages/php/
+language: php
+
+# Specify when Travis should build.
+branches:
+  only:
+    - master
+    - /^release-v.*$/
+
+cache:
+  directories:
+    - $HOME/.composer/cache
+    - ./vendor
+
+matrix:
+  include:
+    - php: '5.3'
+      env: WP_VERSION=3.8
+      dist: precise
+    - php: '5.3'
+      env: WP_VERSION=latest
+      dist: precise
+    - php: '5.6'
+      env: WP_VERSION=3.8
+    - php: '7.0'
+      env: WP_TRAVISCI=phpcs
+    - php: '7.0'
+      env: WP_VERSION=latest
+    - php: '7.3'
+      env: WP_VERSION=latest
+    - php: '7.4'
+      env: WP_VERSION=latest
+    - php: '7.4'
+      env: WP_VERSION=nightly
+  fast_finish: true
+
+# Prepare your build for testing.
+# Failures in this section will result in build status 'errored'.
+before_script:
+  # Turn off Xdebug. See https://core.trac.wordpress.org/changeset/40138.
+  - phpenv config-rm xdebug.ini || echo "Xdebug not available"
+  - export OG_DIR="$(pwd)"
+
+  - export PATH="$HOME/.composer/vendor/bin:$PATH"
+
+  # Couple the PHPUnit version to the PHP version.
+  - |
+    case "$TRAVIS_PHP_VERSION" in
+      7.*)
+        echo "Using PHPUnit 6.1"
+        composer global require "phpunit/phpunit=6.1.*"
+        ;;
+      *)
+        echo "Using PHPUnit 4.8"
+        composer global require "phpunit/phpunit=4.8.*"
+        ;;
+    esac
+
+  - |
+    if [[ "$WP_TRAVISCI" == "phpcs" ]] ; then
+      # Composer Install
+      travis_retry composer install
+      export PATH=$PATH:`pwd`/vendor/bin/
+    fi
+
+  - phpenv rehash
+
+  # For debugging.
+  - pwd
+  - which phpunit
+  - phpunit --version
+  - echo $PATH
+
+# Run test script commands.
+# Default is specific to project language.
+# All commands must exit with code 0 on success. Anything else is considered failure.
+script:
+  # Search for PHP syntax errors.
+  #
+  # Only need to run this once per PHP version.
+  - |
+    if [[ "$PHP_LINT" == "1" ]] ; then
+      find . -type "f" -iname "*.php" -not -path "./vendor/*" | xargs -L "1" php -l
+    fi
+
```

Comment:
Doesn't look like we're ever setting PHP_LINT to true, so this can probably be omitted, unless I'm missing something.